test/controller/objecting.entity.name.controller.spec.unit.ts

```diff
@@ -88,6 +88,22 @@ describe("objecting entity name tests", () => {
     expect(response.status).toEqual(200);
     expect(response.text).toContain(ENTER_FULL_NAME);
     expect(response.text).not.toContain(SELECT_TO_DIVULGE);
+    expect(response.text).toContain("value=\"yes\" checked");
```

Comment:
could you add a not to contain 'no' and similar in the other test for 'yes'

Reply:
Already done.

Reply:
I think Mark means adding
`expect(response.text).not.toContain("value=\"no\" checked");`
In this test, and the same for 'yes' in the test below.
Checking that it is just "Yes" checked here and "No" is not checked

Reply:
yes , we are checking if yes is checked, but the test will pass if both yes and no are checked

Reply:
I did know you could check both I'll have to try that out - updated the tests